test/functional/fixtures/regression/gh-2011/test.js

```diff
@@ -0,0 +1,36 @@
+const path           = require('path');
+const createTestCafe = require('../../../../../lib');
```

Comment:
Please use import instead of require

Reply:
it's a not a testcafe test file. We use CommonJS approach for test mocha files